playground/src/App.tsx

```diff
@@ -0,0 +1,200 @@
+import lzstring from "lz-string";
+import Editor, { useMonaco } from "@monaco-editor/react";
+import { MarkerSeverity } from "monaco-editor/esm/vs/editor/editor.api";
+import { useEffect, useState, useCallback } from "react";
+
+import init, { Check, check } from "./pkg/ruff.js";
+import { AVAILABLE_OPTIONS } from "./options";
+import { Config, getDefaultConfig, toRuffConfig } from "./config";
+
+function restoreConfigAndSource(): [Config, string] {
+  let value = lzstring.decompressFromEncodedURIComponent(
+    window.location.hash.slice(1)
+  );
+  let config = {};
+  let source = "print(1 + 2)";
```

Comment:
Is this the default editor text? (If so, extract to a top-level constant?)

Reply:
It's the default, yes. I extracted it out. We probably want to pick something more exciting though. :D